### PR TITLE
Add: Mobile responsive Members page

### LINKED ISSUE #3479: Create mobile responsive Members page
## Specification

Relates to: https://github.com/JoinColony/colonyDapp/issues/3474

### Story

When viewing the Members page on my mobile, I would like the dapp to adjust to fit my mobile screen. 

### Description 

This issue addresses Screen 5 of the [Figma design](https://www.figma.com/file/DjVCPq7LpkjEMEfQZtKgBdC1/Style-Guide?node-id=0%3A1). 

### Design

image

### This issue is complete when...

- [ ] The members page fits on screens at or less than `768px` in width

### PR DESCRIPTION
**Pull request by willm30**
## Description

This PR is for a mobile responsive Members page, per screen 5 of the [Figma design](https://www.figma.com/file/DjVCPq7LpkjEMEfQZtKgBdC1/Style-Guide).

**New stuff** ✨

* Extracted `ColonyHomeInfo` from `ColonyHomeLayout` for reuse in `ColonyMembers`

**Changes** 🏗

*  `ColonyMembers`, `Members`, `MembersListItem` and related components styled for mobile.

## Screenshots

**_Members page_**
mem-mob-page

**_Teams dropdown_**
teams-menu

**_Kebab_**
kebab-mob

**_Info popover_**
info-popover-mob

**_Pagination set to 10 members_**
mem-mob-page

Resolves  #3479

### CODE REVIEW

**Review by arrenv — CHANGES_REQUESTED** [review]
Nice one @willm30.

Regarding the branch naming, it is good to have the issue you're resolving in the branch name as of the Git Workflow notion doc. For me at least, it helps with finding the right branch when checking it out. As of the screenshot.

Screenshot 2022-06-24 110404

I think this PR needs to be rebased on the Contributors and Watchers branch, as there are a lot of changes to the members page on that branch.

Regarding this one.

There needs to be more margin on the right side of the kebab menu in each member item, you could match the left side. It will make it easier to touch on mobile screens.

90408283-fad1-45e8-a02a-437b1d0eb7c9

I like what you did with the permissions, however, this kebab menu has been added in the Contributors and Watchers branch, so, this is going to clash. I think permissions just needs to be hidden on mobile altogether, as permissions appears in the avatar popover anyway. Although, this is also broken.

19c0daec-8033-49ec-8776-ec951dc2c876

**Review by rdig — CHANGES_REQUESTED** [review]
I found mostly what Arren found, with one extra, as the background for the Members page should be all white _(as in the specs)_. It will help ensure better readability.

Specs:
Screenshot from 2022-06-24 16-36-52

Current:
Screenshot from 2022-06-24 15-54-37

File: src/modules/dashboard/components/Members/MembersTitle.css [hunk 1]
```diff
@@ -88,3 +90,32 @@
...
+
+  .titleLeft button[class*="CreateDomainButton"] {
+    justify-content: flex-start;
+    margin-left: 10px;
+    gap: 10px;
+  }
+
+  .titleLeft div[class*="CreateDomainButton"] {
+    position: static;
+  }
```
Comment (willm30) [inline src/modules/dashboard/components/Members/MembersTitle.css:120]: Styling the `ColonyDomainSelector`

File: src/modules/dashboard/components/Members/MembersTitle.tsx [hunk 2]
```diff
@@ -78,66 +88,78 @@ const MembersTitle = ({
...
 
+  const isMobile = useMediaQuery({ query });
   return (
     <div className={styles.titleContainer}>
       <div className={styles.titleLeft}>
         <Heading
           text={MSG.title}
           appearance={{ size: 'medium', theme: 'dark' }}
+          textValues={{ isMobile }}
         />
```
Comment (willm30) [inline src/modules/dashboard/components/Members/MembersTitle.tsx:99]: All of the below is just adding the `ColonyDomainSelector` when  `isMobile` is true

**Comment by willm30** [comment]
@arrenv Apologies about the branch name, I will be conscious of this in future branches!

I've rebased the `feature/dapp-mobile-responsive` branch from `contributors-vs-watchers` so this branch is now in sync, and I've addressed the issues in your comment re the info popover. 

@rdig I also fixed the mobile colour scheme on `feature/dapp-mobile-responsive` so this page is now in white as it should be!

**Review by arrenv — CHANGES_REQUESTED** [review]
It is looking better, nice one!

Just a couple of small things, the kebab icons could be a little bigger on mobile, and the menu from the kebab menu should have some padding.

download (20)

**Comment by willm30** [comment]
@arrenv Padding added and kebab menu icon enlarged!

**Review by rdig — APPROVED** [review]
Good with this one as well. Nice job

Screenshot from 2022-06-29 15-05-34

**Review by arrenv — APPROVED** [review]
@willm30 looking good, I'm approving, although there is a little something weird with the dropdown padding. Particularly when logged in vs not being logged in, they are not the same. For example:

Logged in:

download (22)


Not logged in:

download (23)

**Comment by willm30** [comment]
> @willm30 looking good, I'm approving, although there is a little something weird with the dropdown padding. Particularly when logged in vs not being logged in, they are not the same. For example:
> 

Great spot, I have fixed.

**Pull request merged**